[Student Learning] Make show/hide toolbox in App Lab Design Mode tab navigable

levadadenys opened the pull request:
<!--
  A summary of the change, including any relevant background, motivation, and context.
  If relevant, include a description, screenshots, and/or video of the existing and new behavior.
-->

### Make show/hide toolbox in App Lab Design Mode tab
Make the show/hide chevron tab-navigable

(On navigate)
![Screenshot from 2023-01-23 13-50-27](https://user-images.githubusercontent.com/22244040/214032826-a4d5d437-72ed-43dd-956b-57bd11e05680.png)

(On hover)
![Screenshot from 2023-01-27 16-38-26](https://user-images.githubusercontent.com/22244040/215118484-6528e3e3-65ef-4f8e-9707-c5fe01033f9b.png)


## Links

<!--
  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
-->
- jira ticket: [SL-440](https://codedotorg.atlassian.net/browse/SL-440?atlOrigin=eyJpIjoiNzY1ZmIzYTlkNTA0NDJmN2IxNjY0YmZlMGY2YjFkNzkiLCJwIjoiaiJ9)
<!--
- spec: []()

-->

## Testing story

<!--
  Does your change include appropriate tests?
  If so, please describe how the tests included in this PR are sufficient.
  If not, please explain why this change does not need to be tested.
-->

<!-- Other aspects to consider. Delete any sections that are not relevant to your change. -->

## Deployment strategy

## Follow-up work

<!--
  List (ideally with Jira links) any clean-up or technical debt that will be addressed in future work.
-->

## Privacy

<!--
  1.	Does this change involve the collection, use, or sharing of new Personal Data?
  2.	Does this change involve a new or changed use or sharing of existing Personal Data?
-->

## Security

<!-- Link to Jira task(s) where sensitive security issues are discussed privately. -->

## Caching

## PR Checklist:

<!--
  The final step! Before you create your PR, double-check that everything is in order.
  Change [ ] to [X] during creation to check boxes.
-->

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

molly-moen reviewed (commented):
What is the hover behavior for this button now? In the screenshot it looks like there may be a border. We prefer to match the previous behavior as much as possible, which is to turn white with no border on hover.

levadadenys commented:
> What is the hover behavior for this button now? In the screenshot it looks like there may be a border. We prefer to match the previous behavior as much as possible, which is to turn white with no border on hover.


Thanks a lot for the comment! The screenshot that was only visible previously was showing the 'navigation' state of the chevron.
However I've found and fixed issue with box shadow on hover. I've added hover screenshot to the PR description, and will attach it to his comment. The branch is also updated.
![Screenshot from 2023-01-27 16-38-26](https://user-images.githubusercontent.com/22244040/215119072-366f703d-9fe2-4fd1-a252-2c0198ff1628.png)
